uint8 dir_command
